modules/core/src/index.js

```diff
@@ -4,7 +4,8 @@ import * as path from './lib/path/path';
 export {path};
 
 // FILE READING AND WRITING
-export {setPathPrefix, getPathPrefix, resolvePath} from './lib/fetch/file-aliases.js';
+export {setPathPrefix, getPathPrefix, resolvePath} from '@loaders.gl/loader-utils';
```

Comment:
This will be eventually deprecated. Add a TODO, will be removed in next major release?

Reply:
I moved it into the section with re-exported utils from loader-utils. 

As for deprecation, these utils are used by luma.gl,  we can decide whether luma.gl should start importing from `loader-utils`, that module is mostly for loaders...